Tests/UnitTests/Networking/Backend/__Snapshots__/BackendPostDiagnosticsTests/iOS17-testPostDiagnosticsPutsDataInDataKey.1.json

```diff
@@ -0,0 +1,28 @@
+{
+  "headers" : {
+    "Authorization" : "Bearer asharedsecret",
+    "content-type" : "application/json",
+    "X-Apple-Device-Identifier" : "5D7C0074-07E4-4564-AAA4-4008D0640881",
+    "X-Client-Build-Version" : "12345",
+    "X-Client-Bundle-ID" : "com.apple.dt.xctest.tool",
+    "X-Client-Version" : "17.0.0",
+    "X-Is-Sandbox" : "true",
+    "X-Observer-Mode-Enabled" : "false",
+    "X-Platform" : "iOS",
+    "X-Platform-Flavor" : "native",
+    "X-Platform-Version" : "Version 17.0.0 (Build 21A342)",
+    "X-Storefront" : "USA",
+    "X-StoreKit2-Enabled" : "false",
+    "X-Version" : "4.0.0"
+  },
+  "request" : {
+    "body" : {
+      "data" : [
+        "{\n  \"properties\": {\"key\": \"value\"},\n  \"timestamp\": \"2024-04-04T12:55:59Z\",\n  \"name\": \"HTTP_REQUEST_PERFORMED\",\n  \"version\": 1\n}",
```

Comment:
Hmm we should send these as json objects, instead of strings. Maybe we need to process it back to an object after all?

Reply:
hmm good point, I hadn't noticed that! Good catch

Reply:
Ended up doing a refactor to make the code similar to Paywall Events, and now they are sent as json with the correct date format. lmkwyt